packages/outpack_server/default.nix

```diff
@@ -5,7 +5,7 @@ in
 rustPlatform.buildRustPackage rec {
   name = "outpack_server";
   src = fetchFromGitHub sources.src;
-  cargoHash = sources.cargoDepsHash;
+  cargoLock.lockFile = "${src}/Cargo.lock";
```

Comment:
See https://github.com/reside-ic/packit-infra/pull/32 for the original conversation about this